CHANGELOG.md

```diff
@@ -1,3 +1,56 @@
+## 1.11.0
+
+This release sets the ground for the new upcoming KVM qemu flavor. It adds support for exporting a pod to an ACI including all modifications. The rkt API service now also supports systemd socket activation. Finally we have diagnostics back, helping users to find out why their app failed to execute.
+
+#### New features
+- KVM: Hypervisor support for KVM flavor focusing on qemu ([#2684](https://github.com/coreos/rkt/pull/2684)). This provides a generic mechanism to use different kvm hypervisors (such as lkvm, qemu-kvm).
+- rkt: add command to export a pod to an aci ([#2889](https://github.com/coreos/rkt/pull/2889)). Adds a new `export` command to rkt which generates an ACI from a pod; saving any changes made to the pod.
+- rkt/api: add `--socket` option to run as a `systemd.socket(5)` service ([#2916](https://github.com/coreos/rkt/pull/2916)). This allows rkt to run as a systemd socket-based unit.
```

Comment:
The pull request was misnamed, it should be "rkt/api: detect when run as a systemd.socket(5) service"